### DISCUSSION TITLE
Subpar quality

### DISCUSSION THREAD

**Discussion by quintendewilde** [deforum-art/sd-webui-text2video#136] [Q&A]
What am I doing wrong to get such bad quality?

I'm using img to video: Woman dancing around
And this is the result. (below)
Is this because it's using stable diffusion 1.5?

I used nateraw sd2 sdv and I get better results.

![frame000032](https://user-images.githubusercontent.com/6031569/233796636-246eb4ed-131f-4265-be56-58fc547b70e3.png)

https://user-images.githubusercontent.com/6031569/233796600-6a8c0269-d0eb-4391-973d-2d78e26c7539.mp4

**Comment by kabachuha** [comment]
What model?

Looks like you're running it on high resolution. Try making it on lower one (like 256x128) and then upscaling it to your high res. It's the same problem as the regular StableDiffusion duplicating stuff when generating on high res without the high. res. fix

**Reply by aivrar** [reply]
Also use your negative prompt maybe like ex. 'text, watermark, copyright, blurry, deformities, extra fingers, extra arm, extra leg, extra eye, no arm, no leg, bad hand, bad face, extra hand, double head, bad feet' etc. playing with that helps too.
